### ISSUE TITLE
[wimp-sim] fix FIXME (Trac #1144)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1138]
in public/wimpsim-reader/I3WimpParams.h

Migrated from https://code.icecube.wisc.edu/ticket/1144
```json
{
    "status": "closed", 
    "changetime": "2015-08-18T09:04:14", 
    "description": "in public/wimpsim-reader/I3WimpParams.h", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1439888654214643", 
    "component": "combo simulation", 
    "summary": "[wimp-sim] fix FIXME", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T22:10:33", 
    "milestone": "", 
    "owner": "mzoll", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150818T08:59:52: 
* mzoll commented:

> Weeeeell this FIXME is something I cannot fix; I do not even know if this fixme is really needed.
> 
> I am no expert in the macro-magic that is going on in some corners of icetray; I have seen this invocation in other places, but it doesn't work for me at this place though. I have no idea why. Maybe some other expert, which had his fingers in the guts of icetray before knows a resolution.
> 
> EXPERT INPUT IS NEEDED


* mzoll changed status from "new" to "accepted"
* mzoll changed owner from "marcel.zoll@fysik.su.se" to "mzoll"

**Comment by icecube-trac** [comment]
Trac update at 20150818T09:04:14: 
* jvansanten commented:

> In changeset 136355:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136355"
> Include the right headers for I3_DEFAULT_NAME. Fixes #1138.
> ```


* jvansanten changed status from "accepted" to "closed"
* jvansanten changed resolution from "" to "fixed"
